contrib/ci/Dockerfile-debian

```diff
@@ -0,0 +1,38 @@
+FROM debian:testing
+
+RUN echo "deb http://ftp.us.debian.org/debian unstable main contrib non-free" >> /etc/apt/sources.list
+RUN echo 'Package: *\n\
+Pin: release a=testing\n\
+Pin-Priority: 900\n\
+\n\
+Package: *\n\
+Pin: release a=unstable\n\
+Pin-Priority: 800\n'\
+ > /etc/apt/preferences
+ > /etc/apt/preferences
+RUN cat /etc/apt/preferences
+RUN echo fubar > /etc/machine-id
```

Comment:
I would think this whole block is probably not needed

Reply:
Could you do me a favour and fix the Debian CI? It's a bit weird to my little Red Hat brain.

Reply:
That was more painful than I expected to get s390x working. But it's there now.

Something is awry in the test suite for Windows still though.